picard/ui/filter.py

```diff
@@ -45,20 +47,23 @@ class Filter(QtWidgets.QWidget):
     def __init__(self, parent=None):
         super().__init__(parent)
         Filter.instances.add(self)
+        self.saved_filters_key = f"{type(parent).__name__}FilterItems"
```

Comment:
Just a minor thing, but for most keys like this in persist we use 'key_Class' format, maybe stick to that.

```suggestion
        self.saved_filters_key = f"filters_{type(parent).__name__}"
```